.github/pull_request_template.md

```diff
@@ -13,18 +13,28 @@ List any other tasks / PRs that are required to be merged alongside this one (e.
 If any of these are not applicable, strikethrough the line `~like this~`. **Do not delete it!**. Let the reviewer decide if you should have done it.
 
 ### Code
-- [ ] I have performed a self review of my own code (including checking issues raised when creating the PR).
-- [ ] I have added/updated unit tests for these changes, and if not I have explained why they are not necessary.
-- [ ] I have commented my code in any hard-to-understand or hacky areas.
-- [ ] I have handled all new warnings generated by the compiler or IDE.
-- [ ] I have rebased onto the target branch (usually main).
+<input type="checkbox"/> 
+<label>I have performed a self review of my own code (including checking issues raised when creating the PR).</label> <br>
+<input type="checkbox"/> 
+<label>I have added/updated unit tests for these changes, and if not I have explained why they are not necessary.</label> <br>
+<input type="checkbox"/> 
+<label>I have commented my code in any hard-to-understand or hacky areas.</label> <br>
+<input type="checkbox"/> 
+<label>I have handled all new warnings generated by the compiler or IDE.</label> <br>
+<input type="checkbox"/> 
+<label>I have rebased onto the target branch (usually main).</label> <br>
```

Comment:
in the PR, what happens if you try strikethrough these?
e.g if you surround by ~